src/main/java/dev/galacticraft/mod/GalacticraftMixinPlugin.java

```diff
@@ -41,6 +44,9 @@ public String getRefMapperConfig() {
 
     @Override
     public boolean shouldApplyMixin(String targetClassName, String mixinClassName) {
+        if ((mixinClassName.contains("ExplosionDamageCalculatorMixin") || mixinClassName.contains("ServerPlayerGameModeMixin")) && !OMNISHAPE_LOADED) {
+            return false;
+        }
```

Comment:
A bit nitpicky but I think this is clearer.
```suggestion
        if ((mixinClassName.contains("ExplosionDamageCalculatorMixin") || mixinClassName.contains("ServerPlayerGameModeMixin"))) {
            return OMNISHAPE_LOADED;
        }
```
It would probably be a good idea to include the `omnishape.` package prefix in the test (since the mixins are named after their targets)